plugins/modules/database/saphana/hana_query.py

```diff
@@ -99,27 +99,27 @@
 
 RETURN = r'''
 
-stdout:
+query_result:
     type: str
     description: A JSON string of the returned values from the SQL queries.
     returned: on success
-    sample: '{ "AVG_TIME_S": "0.65", "CHECK_ACTION": "CHECK", "CHECK_PROCEDURE_NAME": "CHECK_TABLE_CONSISTENCY", "ERROR_DETAILS": "0" }'
+    type: list
+    elements: dict
```

Comment:
```suggestion
    elements: list
```
This is more likely a list of lists, from your code.